src/sdkRuntimeModels.ts

```diff
@@ -161,51 +159,36 @@ export interface SDKProduct {
     Attributes?: Record<string, unknown> | null;
 }
 
+// https://go.mparticle.com/work/SQDSDKS-6949
 export interface MParticleWebSDK {
     addForwarder(mockForwarder: MPForwarder): void;
-    _IntegrationCapture: IntegrationCapture;
     IdentityType: valueof<typeof IdentityType>;
     CommerceEventType: valueof<typeof CommerceEventType>;
     EventType: valueof<typeof EventType>;
     PromotionType: valueof<typeof PromotionActionType>;
-    _Identity: IIdentity;
     Identity: SDKIdentityApi;
     Logger: SDKLoggerApi;
-    MPSideloadedKit: IMPSideloadedKit;
-    _APIClient: any; // TODO: Set up API Client
-    _CookieSyncManager: ICookieSyncManager;
-    _Store: IStore;
-    _Forwarders: any;
-    _Helpers: SDKHelpersApi;
-    _Events: IEvents;
-    _Ecommerce: IECommerce;
-    config: SDKInitConfig;
-    _ServerModel: IServerModel;
-    _SessionManager: ISessionManager;
-    _Consent: SDKConsentApi;
     Consent: SDKConsentApi;
-    _NativeSdkHelpers: INativeSdkHelpers;
-    _Persistence: IPersistence;
-    _preInit: any; // TODO: Set up API
-    _instances?: Dictionary<MParticleWebSDK>;
-    _isTestEnv?: boolean;
     _resetForTests(
         MPConfig?: SDKInitConfig,
         keepPersistence?: boolean,
-        instance?: MParticleWebSDK
+        instance?: IMParticleWebSDKInstance,
     ): void;
+    configurePixel(config: IPixelConfiguration): void;
     endSession(): void;
-    identifyRequest: IdentityApiData;
     init(apiKey: string, config: SDKInitConfig, instanceName?: string): void;
     _getActiveForwarders(): MPForwarder[];
+    _getIntegrationDelays(): IntegrationDelays;
```

Comment:
ditto to the above comment - this is probably just because we have a setter, and so added a getter as well, but i don't think it's used anywhere

Reply:
Since it's technically "private" or "internal" can we kill it?

Reply:
i'd say no to killing just because it's publicly accessible...who knows if someone is actively using it :/